driver/kubernetes/driver.go

```diff
@@ -113,7 +114,7 @@ func (d *Driver) wait(ctx context.Context) error {
 		select {
 		case <-ctx.Done():
 			return ctx.Err()
-		case <-time.After(time.Duration(100+try*20) * time.Millisecond):
```

Comment:
I think we should keep the backoff logic

Reply:
It will be ok for you if I will just change `100` → `300`

Reply:
> I'd be interested to know why the backoff is better when the nodes don't spin up in time.

@SimonJPegg Don't think you would want to hammer the API every `Xms` but instead backoff on each retry. Although looking at the retry logic it seems fair to not keep it.

> It will be ok for you if I will just change `100` → `300`?

I think it's fine to keep `100` as default but might be better to use a duration, see https://github.com/docker/buildx/pull/2457#discussion_r1601536109